src/components/header/Header.tsx

```diff
@@ -354,21 +396,113 @@ const Header = () => {
                 borderRadius: "10px",
                 outline: "none",
                 fieldset: { border: "none", outline: "none" },
+               
               }}
             >
               <SearchIcon sx={{ ml: { xl: "10px" } }} />
-              <TextField
-                placeholder="Find your perfect deal"
+           
+             
+              <Autocomplete
+              className="xyz"
+              value={searchValue}
+              open={openListBox}
+              disableClearable
+              onFocus={()=>{
+                if(searchValue){
+                  setOpenLIstBox(true)
+                }
+              }}
+
+              PopperComponent={(props) => (
+                <Popper {...props} style={{ width: "28.75rem" }}>
+                  {props.children}
+                </Popper>
+              )}
+              onClose={()=>setOpenLIstBox(false)}
+               
+                freeSolo
+                id="country-select-demo"
                 sx={{
-                  height: "100%",
-                  width: "25.4rem",
-                  border: "none",
-                  // bgcolor:"blue",
-                  justifyContent: "center",
-                  fieldset: { border: "none", outline: "none" },
+                  width: "24.7rem",
+                  "MuiAutocomplete-loading": { display: "none" }, "&:ul":{background:"pink", width:"300%"}
```

Comment:
Why need width `300%`

Reply:
Sir I create that class just for testing purpose its not needed now so i will remove it.